studies/BraveAIChatDefaultModelStudy.json5

```diff
@@ -75,11 +114,16 @@
             name: 'default_model',
             value: 'chat-qwen',
           },
+          {
+            name: 'default_premium_model',
+            value: 'chat-claude-sonnet',
+          },
         ],
       },
     ],
     filter: {
       min_version: '133.1.77.19',
+      max_version: '137.1.80.62',
```

Comment:
Do we want a max version here?

Reply:
nvm, i see that you want to rely on automatic model selection in 1.80.63

Reply:
Yes, this is intentional. "Stopped the current study for versions with automatic model."